dotnet/samples/Concepts/ChatCompletion/ChatHistoryReducers/ChatHistoryExtensions.cs

```diff
@@ -19,27 +19,29 @@ internal static class ChatHistoryExtensions
     /// <remarks>
     /// For simplicity only a single system message is supported in these examples.
     /// </remarks>
-    internal static ChatMessageContent? GetSystemMessage(this ChatHistory chatHistory)
+    internal static ChatMessageContent? GetSystemMessage(this IReadOnlyList<ChatMessageContent> history)
     {
-        return chatHistory.FirstOrDefault(m => m.Role == AuthorRole.System);
+        return history.FirstOrDefault(m => m.Role == AuthorRole.System);
```

Comment:
nit. I think a list of chatmessages should still be considered a chatHistory for the var name. We may have other places where we refer to chatHistory, just for consistency.

```suggestion
        return chatHistory.FirstOrDefault(m => m.Role == AuthorRole.System);
```

Reply:
Makes sense, updated!